src/lib/barrier/KeyMap.cpp

```diff
@@ -735,7 +735,7 @@ KeyMap::keyForModifier(KeyButton button, SInt32 group,
 
     // find a key that generates the given modifier in the given group
     // but doesn't use the given button, presumably because we're trying
-    // to generate a KeyID that's only bound the the given button.
+    // to generate a KeyID that's only bound the given button.
```

Comment:
Duplicate word handling is a relatively new thing for me, but I think this is right.